libraries/Microsoft.Bot.Builder.LUIS/LuisRecognizerMiddleware.cs

```diff
@@ -17,13 +18,16 @@ namespace Microsoft.Bot.Builder.LUIS
     public class LuisRecognizerMiddleware : IMiddleware
     {
         public const string LuisRecognizerResultKey = "LuisRecognizerResult";
-        private readonly IRecognizer _luisRecognizer;
+        public const string LuisTraceEventName = "LuisTraceInfo";
+        public const string Obfuscated = "****";
+        private readonly ILuisRecognizer _luisRecognizer;
+        private readonly ILuisModel _luisModel;
+        private readonly ILuisOptions _luisOptions;
 
         public LuisRecognizerMiddleware(ILuisModel luisModel, ILuisRecognizerOptions luisRecognizerOptions = null, ILuisOptions luisOptions = null)
         {
-            if(luisModel == null)
-                throw new ArgumentNullException(nameof(luisModel));
-
+            _luisModel = luisModel ?? throw new ArgumentNullException(nameof(luisModel));
```

Comment:
Please add Unit Tests that validate excpetions are thrown. on these parameters.

Reply:
Done